README.md

```diff
@@ -6,33 +6,66 @@
 [![Discourse status](https://img.shields.io/discourse/https/meta.discourse.org/status.svg?style=for-the-badge)](https://forums.habitat.sh)
 [![Open Source Helpers](https://www.codetriage.com/habitat-sh/habitat/badges/users.svg)](https://www.codetriage.com/habitat-sh/habitat)
 
-Want to try Habitat? [Get started here](https://www.habitat.sh/learn/).
+<img src="https://github.com/habitat-sh/habitat/blob/master/www/source/images/habitat-logo.png" width="150">
 
-Habitat is an application automation framework that allows you to build applications that have automation built-in. This provides modern applications that:
+[Habitat](http://habitat.sh) is open source software that creates platform-independent build artifacts and provides built-in deployment and management capabilities.
 
-* Provide repeatable builds
-* Run from single, immutable assets
-* Allow for runtime configuration for multiple deployment scenarios
-* Are agnostic to operating environment (works the same on bare metal, virtualization, containers, PaaS)
-* Provide idempotent behavior (the same inputs to the same asset provide the same outcome)
-* Provide convergent behavior (each service makes progress towards the correct behavior in the face of failure)
-* Expose promises to those who rely on it
-* Provide a low barrier to entry
-* Are language agnostic
+The goal of Habitat is to allow you to automate your application behavior when you create your application, and then bundle your application with the automation it needs to behave with the correct run time behavior, update strategies, failure handling strategies, and scaling behavior, wherever you choose to deploy it.
```

Comment:
I wonder if the message here could be more tightly aligned with "build, deploy and manage."

Reply:
I think we will deemphasize the three verbs on the upcoming new homepage(https://chef.zoom.us/j/186686947), but I am aware that in the new video we use them. This was drafted by @tashimi , what do you think, Tasha?

Reply:
sounds good!